draft-ounsworth-pq-composite-sigs.mkd

```diff
@@ -293,6 +295,8 @@ Signature Generation Procedure:
 ~~~
 {: artwork-name="alg-composite-sig-gen"}
 
+Since recursive composite public keys are banned in section {{sec-composite-pub-keys}}, it is safe to assume that the private keys K1, K2, .., Kn will not contain composite types.
```

Comment:
"Assuming" is not safe, we should add language so that it rejects. (either here or during the public key check?)

> ... verifiers should fail if one or more of K1, K2, .., Kn contain a CompositePublicKey.


Do the same in the sig verif section

Reply:
Also change "banned" to "disallowed"

Reply:
Max: "when verifying the composite signature, if the compositecrypto OID is used in one of the "composing" signatures, the relying party MUST reject the signature"